docs/guides/first-app.md

```diff
@@ -3,33 +3,37 @@
 import TabItem from '@theme/TabItem';
 import Tabs from '@theme/Tabs';
 
-:::caution Work in Progress
-Not all languages in this guide are up to date.
-:::
-
 ## Introduction
 
-This guide walks through defining a permission system for a very simple blog application.
-
-Each section demonstrates how to use the API to perform the required step and the [final section] demonstrates these APIs calls combined into one end-to-end integrated application.
+This guide walks through the steps required to deeply integrate an application with SpiceDB.
+Not all software requires this level of integration, but it is preferable for greenfield applications or applications that are central in an architecture with multiple services.
 
-[final section]: #end-to-end-example
+Instead of introducing an unfamiliar example app and altering various locations in its code, this guide is written such that each step is a standalone snippet of code that demonstrates an integration point and finding where those points exist in your codebase is an exercise left to the reader.
 
 ## Prerequisites
 
-- An [Authzed] account
-- An [API Token] with _admin_ access to a new Permissions System
+- An [Authzed] permission system **OR** a running instance of [SpiceDB][SpiceDB]
```

Comment:
the preview of `[SpiceDB][SpiceDB]` looks correct but...why?

Reply:
This is another syntax for markdown. Instead of parens that contain a link, you can put square brackets and give the link an explicit name that you can use later.

```md
[mylink][its-name] is cool

[its-name]: coolwebsite.com
```

Reply:
TIL 👍